practice1.js

```diff
@@ -14,31 +13,40 @@ btn.addEventListener('click', () =>
   // 値を代入する変数を宣言
   let val = '';
 
-  // fizzとbuzzの値に空欄,文字列,小数が存在した場合、outputの中身を空にして、注意文を表示する
-  if (!fizz || fizz.match(/^\D+$/) || fizz.match(/^-?[0-9]+\.[0-9]+$/) || !buzz || buzz.match(/^\D+$/) || buzz.match(/^-?[0-9]+\.[0-9]+$/)) {
+  // fizzとbuzzの値が空ならば、outputの中身を空にして注意文を表示する
+  if (!fizz || !buzz) {
+    output.innerHTML = '';
+    output.innerHTML = '<p>整数値を入力してください</p>'
+
+    // fizzとbuzzの値が数字以外ならば、outputの中身を空にして注意文を表示する
+  } else if (fizz.match(/^\D+$/) || buzz.match(/^\D+$/)) {
+    output.innerHTML = '';
+    output.innerHTML = '<p>整数値を入力してください</p>'
+
+  // fizzとbuzzの値が小数ならば、outputの中身を空にして注意文を表示する 
+  } else if (fizz.match(/^-?[0-9]+\.[0-9]+$/) || buzz.match(/^-?[0-9]+\.[0-9]+$/)) {
     output.innerHTML = '';
     output.innerHTML = '<p>整数値を入力してください</p>'
 
   // fizzとbuzzの値が整数値の場合
   } else {
-
     // outputの中身を空にする
     output.innerHTML = '';
 
       // 変数iを宣言し、iに1を代入。iが100未満なら+1を繰り返す
       for (let i = 1; i < 100; i++) {
 
         // iがfizzとbuzzの倍数なら、valに文字列とiを代入
-        if (i % fizz == 0 && i % buzz == 0) {
-          val = 'FizzBuzz' + i;
+        if (i % fizz === 0 && i % buzz === 0) {
+          val = 'FizzBuzz' + ' ' + i;
```

Comment:
[テンプレート文字列](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/template_strings)を利用して以下の通り書いてもいいかなと思いました
```suggestion
          val = `FizzBuzz ${i}`;
```